stdlib/socket/from.go

```diff
@@ -0,0 +1,309 @@
+// This source gets input from a socket connection and produces tables given a decoder.
+// This is a good candidate for streaming use cases. For now, it produces a single table for everything
+// that it receives from the start to the end of the connection.
+package socket
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"net"
+	"strings"
+	"time"
+
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/csv"
+	"github.com/influxdata/flux/execute"
+	"github.com/influxdata/flux/memory"
+	"github.com/influxdata/flux/plan"
+	"github.com/influxdata/flux/semantic"
+	"github.com/influxdata/flux/values"
+	"github.com/pkg/errors"
+)
+
+const FromSocketKind = "fromSocket"
+
+type RawDecoder struct {
```

Comment:
What does RawDecoder do? What format is it decoding?

Reply:
I'll put docs above it.

Reply:
What is the purpose of the decoder? Just a simple way to test the source?

Reply:
Nope, it takes raw input from the reader and puts it into a `_value` column. I put the doc above it now.
It is a way to get raw messages from the socket, without any specified schema as the CsvDecoder does.

Reply:
OK, in that case I think we should change a few things. 

* It should be moved to its own package and have unit tests for its decoding methods.
* It should not add the _start and_stop columns, that is the function of the range transformation and it doesn't make sense here

Reply:
The influxdb.from adds _start and _stop because it is acting as the range transformation since it gets pushed down.

Reply:
For the first point 👍 

For the second, how `range` work with something that doesn't have timestamps attached?
The same for `window`.

Reply:
Oh ok, I misunderstood your point. So, _time is ok, but not _start and _stop.